_includes/js/queries.md

```diff
@@ -57,6 +57,27 @@ You can skip the first results by setting `skip`. In the old Parse hosted backen
 query.skip(10); // skip the first 10 results
 ```
 
+If you want to know the total number of rows in a table satisfying your query, for  e.g. pagination purposes - you can use `withCount` (`false` by default). 
+
+**Note:** Enabling this flag will change the structure of response, see the example below.
```

Comment:
I thought it was true by default? You can just use query.withCount()?

Reply:
If you want to know the total number of rows in a table satisfying your query, for  e.g. pagination purposes - you can use `withCount()` (optionally accepts a boolean, defaults to `true` when called without arguments or `undefined`).
 
Will this be more clear?

Reply:
(optionally accepts a boolean, defaults to true)  should be fine

Reply:
can't this be mis-interpreted like it's an always enabled by default on every query?

Reply:
@TomWFox What do you think?

Reply:
I agree with @dplewis, it's referencing that particular method not querying in general

Reply:
Alternatively, you could not mention the optional boolean as its not strictly relevant to this explanation.

Reply:
right, jsdoc will show these details anyway when it comes to coding

Reply:
@dplewis are you happy with that change?